HW2/P4/P4.txt

```diff
@@ -0,0 +1,10 @@
+# Your discussion here
+1 thread
+13.5527689457 seconds for 10 filter passes.
+
+2 threads
+13.3241732121 seconds for 10 filter passes.
+
+4 threads
+13.2391269207 seconds for 10 filter passes.
+
```

Comment:
I do not see improvements after using threading in python in my program as well. Is it because of the python threads are actually executed in serial?